.github/workflows/fix-conflicts.yml

```diff
@@ -0,0 +1,27 @@
+name: Solve conflicts
+
+on:
+  pull_request:
+    types:
+      - synchronize
```

Comment:
what does this `synchronize` thing mean? I couldn't find anything in github actions docs

Reply:
`syncronize` means that this action will be trigged each time that a push occurs to PR, but it will only run if is there any label `has conflicts`
Here are some references:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
https://frontside.com/blog/2020-05-26-github-actions-pull_request/